docs/tutorial.rst

```diff
@@ -348,7 +350,8 @@ For more on this templating, see :ref:`templates`
             - |
               AWS_ACCESS_KEY_ID={{ .Secrets.aws.Data.aws_access_key_id | toString }}         \
               AWS_SECRET_ACCESS_KEY={{ .Secrets.aws.Data.aws_secret_access_key | toString }} \
-              aws s3 cp /var/log/time.log {{ .ConfigMaps.location.Data.path | quote }}
+              aws s3 cp /var/log/time.log {{ .ConfigMaps.location.Data.path | quote }} \
+              --endpoint {{ .Profile.Location.Endpoint }}
```

Comment:
If we are changing the command just for minio we should have updated command separately in Note section.